bw2parameters/ecoinvent_units.txt

```diff
@@ -0,0 +1,17 @@
+# default "ton" is interpreted as "short ton" (2000 pounds)
+ton = 1000 kg
+
+# ecoinvent specific stuff
+unit = [unit]
+person = [person] = _ = guest
+night = [time]
+EUR2005 = [currency]
+kilo = 1000
```

Comment:
I thought that these prefixes were already defined in Pint, but I guess I was dreaming? But if we need to define prefixes that this shouldn't be in "ecoinvent specific stuff".

Reply:
Yes and no. Pint can interpret stuff like this:
``` python
>>> from pint import UnitRegistry
>>> ureg = UnitRegistry()
>>> ureg("kilobecquerel").to("becquerel")
<Quantity(1000.0, 'becquerel')>
```
Unfortunately, ecoinvent does not use "kilobecquerel". They write "kilo Becquerel" and that causes two errors in `pint`: it tries to interpret "kilo" (which it doesn't know out of the box) and it tries to interpret "Becquerel" (which it doesn't recognize because of the capital letter). So these lines really *are* necessary just to make ecoinvent work.